source/code-generation/query-converter/test-converted-queries/install-query-runner.txt

```diff
@@ -0,0 +1,94 @@
+.. _rm-install-query-runner:
+
+====================
+Install Query Runner
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To use Query Converter Test Bench, Relational
+Migrator must have access to the Query Runner. 
+Query Runner is a Docker container with 
+all the dependencies needed to test your converted 
+queries.
+
+About this Task
+---------------
+
+- Query Runner operates on the ``http`` protocol and 
+  is not encrypted. All data is processed within the 
+  container and no connection details or sensitive data 
+  is exposed during the Query Runner connection process.
+
+- Query Runner can run either locally or remotely.
+
+Before you Begin
+----------------
+
+Install `Docker <https://www.docker.com/get-started/>`__.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Get the Query Runner Image
+      
+      .. code-block:: sh
+
+         docker pull public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+   .. step:: Run the Image as a Container
+
+         - To run the container locally:
+         
+            .. code-block:: sh
+
+               docker run --name relational-migrator-query-runner -p 6080:6080 public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+         - To run the container remotely:
```

Comment:
You still need to launch the container - currently the steps imply the `docker run` step is only if it's local. If my suggestion above makes sense we can remove the localhost IP from the remote version.

Reply:
Good point, I added an additional code block and some content summarizing this point with the original network exposed docker `6080:6080` run command.